ibm/resource_ibm_hpcs.go

```diff
@@ -709,6 +709,9 @@ func expandHSMConfig(d *schema.ResourceData, meta interface{}) tkesdk.HsmConfig
 		}
 		hsmConfig.Admins = admins
 	}
+	if f, ok := d.GetOk("failover_units"); ok {
```

Comment:
Please add this if block [here](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/d7ffbf3bc680c4a6dc6efa42bc17d66d85e19bc8/ibm/resource_ibm_hpcs.go#L664).. outside `expandHSMConfig` function..

This function is used in Delete functions as well.. So please have this outside